### ISSUE TITLE
L2 regularization strength for few-shot linear probe 

### ISSUE THREAD

**Issue by constanceferragu** [openai/CLIP#128]
Hello, could you please explain how you determined the L2 regularization strength for the few-shot linear probe? Did you use a hyperparameter sweep on the validation sets, and if so, how did you do that given that there isn't a lot of available training data in few-shot learning? Thank you in advance!

**Comment by jongwook** [comment]
Across all compared models, we used the performance on same validation set that we used for the full-data linear probe. It's not truly few-shot learning in this sense, although we hoped to find a consistent benchmark to fairly compare against the models (Fig. 6), and also have a means to plot the interpolation line to get a sense of the effective robustness (Figure. 15).
